pkilint/etsi/en_319_412_5.py

```diff
@@ -332,17 +352,25 @@ def retrieve_qualified_statement_id(cls, node):
     def __init__(self, certificate_type: etsi_constants.CertificateType):
         allowances = {}
 
-        if certificate_type in etsi_constants.EU_QWAC_TYPES:
+        if certificate_type in etsi_constants.EU:
+            # Table 2: 4.2.1
             allowances[en_319_412_5.id_etsi_qcs_QcCompliance] = Rfc2119Word.MUST
+            # Table 2: 4.2.4
             allowances[en_319_412_5.id_etsi_qcs_QcCClegislation] = Rfc2119Word.MUST_NOT
-            allowances[en_319_412_5.id_etsi_qcs_QcType] = Rfc2119Word.MUST
+            # Table 2: 4.2.2
+            if certificate_type in etsi_constants.EU_SSCD:
+                allowances[en_319_412_5.id_etsi_qcs_QcSSCD] = Rfc2119Word.MUST
+
+            # Table 2: 4.2.3 (QWAC is Annex IV)
+            if certificate_type in etsi_constants.EU_QWAC_TYPES:
+                allowances[en_319_412_5.id_etsi_qcs_QcType] = Rfc2119Word.MUST
+                # PR Question: Table 2, 4.2.2 only defines MUST, is the MUST_NOT also from 412-5 somewhere?
+                allowances[en_319_412_5.id_etsi_qcs_QcSSCD] = Rfc2119Word.MUST_NOT
+
         elif certificate_type in etsi_constants.NON_EU_QWAC_TYPES:
-            allowances[en_319_412_5.id_etsi_qcs_QcCompliance] = Rfc2119Word.MUST
+            # PR Question: Is this from 415_5.qcs-4.2? Needs different classifier?
             allowances[en_319_412_5.id_etsi_qcs_QcCClegislation] = Rfc2119Word.MUST
```

Comment:
I was wondering what the source was for this rule. I couldn't really find it other than the reference in 412-5 QCS 4.2. If so, would it need a different source in the  validation finding?